### ISSUE TITLE
Syncpad for Simplenote installation broken (Windows, Chrome)

### ISSUE THREAD

**Issue by soupnap** [shatting/simplenote#12]
It has become impossible to install the extension for new users. This problem is consistent across platforms. Trying to install the extension results in this error message: "Could not load extension icon 'imanges/icon_96.png"

http://i.imgur.com/qrFMYSO.jpg

**Comment by K9SIB** [comment]
I can confirm this error as I just tried to install the extension on my Windows 7 box.  It worked earlier today when I installed it on chromium in Ubuntu though.

**Comment by JorelLatraille** [comment]
Yes it looks like there is a typo in one of the scripts and it is looking for a directory called "imanges" which should be "images"

**Comment by rslee0** [comment]
Can also confirm unable to install on Windows/Chrome due to this error.

**Comment by niu541412** [comment]
yes, it is due to the wrong name

**Comment by thesturs** [comment]
I got the same error...

Windows 7 64bit Chrome Version 31.0.1650.57 m

**Comment by mickeoL** [comment]
it seems the plugin hasnt been updated for a long time?

**Comment by ocorreiododiogo** [comment]
I confirm the error. There is a typo on the image name

**Comment by PerfectFaro** [comment]
Any plans to fix this, or has the extension been abandoned?

**Comment by JessicaC** [comment]
Can the webstore extension be FIXED already??!

**Comment by ghost** [comment]
Same error here...

**Comment by aquilax-jp** [comment]
same error until the latest version of chrome...

**Comment by gsingh93** [comment]
My version fixes the issue and there is a new version uploaded to the chrome store under my account: https://github.com/gsingh93/simplenote

The link on that page will take an hour or two from posting this to work. Also, I'm not 100% sure my fix for the saving issue was correct, so if someone who knows JavaScript can verify [this commit](https://github.com/gsingh93/simplenote/commit/e12f848c607588610e89072736ae662689ae0d96) makes sense, that would be appreciated.

**Comment by matthewm1970** [comment]
I can't speak in general, but I can confirm that it worked fine on my system, with OSX 10.9.4 running Chrome 43.0.2357.81 (64-bit).  Nice job.

**Comment by rslee0** [comment]
It also works for me. Same version of Chrome on Win 8.1. Thanks!
